example/device/framegrabber/getRealSense2Info.cpp

```diff
@@ -50,7 +50,11 @@ int main()
     vpRealSense2 rs;
     rs.open();
     std::cout << "RealSense characteristics:\n" << rs << std::endl;
-    std::cout << "Depth scale: " << std::setprecision(std::numeric_limits<float>::max_digits10) << rs.getDepthScale() << std::endl;
+
+    std::string product = rs.getProductLine();
+
+    if(product.compare("T200") != 0) // If it has depth sensor
```

Comment:
Wondering if there is a way to check if a device has depth support in the `librealsense` library?
Maybe something like this?

https://github.com/IntelRealSense/librealsense/blob/025fccf76803ee6a6e60de9f18ac6193b7ff8597/unit-tests/unit-tests-live.cpp#L5248

Reply:
Yes this would be perfect

Reply:
Will check and modify it.

Reply:
Done.